### PR TITLE
Added future term dates in config file

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323

File: shared/gh/files/config.json [hunk 1]
```diff
@@ -8,147 +8,266 @@
...
+                "name": "lent",
+                "label": "Lent",
+                "start": "2022-01-18",
+                "end": "2022-03-18"
+            },
+            {
+                "name": "easter",
+                "label": "Easter",
+                "start": "2022-04-27",
+                "end": "2022-06-18"
```
Comment (bp323) [inline shared/gh/files/config.json]:
```javascript
"start": "2022-04-26",
"end": "2022-06-17"
```

per spreadsheet

**Comment by bp323** [comment]
One incorrect date per spreadsheet, back to @Coenego for followup

**Comment by Coenego** [comment]
Back to @bp323 after follow-up

**Comment by bp323** [comment]
Verified and merged!

**Pull request merged**